blackout/parserapp/services/parser.py

```diff
@@ -66,59 +70,82 @@ def scrap_data(page_data):
             "time_on": other[7].text
         }
         data.append(j)
-    return data
+    date = soup.find(title="Source Title").text
 
+    # Replace characters from date
+    chars = [" ", ".", ":"]
+    for el in chars:
+        if el != ".":
+            date = date.replace(el, "_")
+        else:
+            date = date.replace(el, "")
+
+    # Following block checks if program started outside of folder parserapp.
+    # It's caused by using django management commands
+    if "parserapp" in os.listdir():
+        prefix = "parserapp/services/"
+        if "pages" not in os.listdir(path="parserapp/services/"):  # Checks does directory "pages" exist and create if not
+            os.makedirs("parserapp/services/pages")
+    else:
+        prefix = None
+        if "pages" not in os.listdir():
+            os.makedirs("pages")
```

Comment:
Make a validator function that rise an error if the program is run externally and just call it here.